Files/UserControls/NavigationToolbar.xaml

```diff
@@ -572,6 +572,7 @@
                     Width="36"
                     Height="32"
                     AutomationProperties.Name="Settings"
+                    AccessKey="O"
```

Comment:
```suggestion
                    AccessKey="I"
```

It's probably better to use a letter that's in the word "Settings".

Reply:
![image](https://user-images.githubusercontent.com/11770760/126912416-33b9e54e-4b53-42c9-be85-8949371b91d4.png)

Could be one of those letters that¡ is not already taken on the main menu? I went with that letter because while testing I pressed again the key and it throws an exception because the settings were already open, so to bypass this I used a key that is already present on settings, so when you open the settings it got overwritten.

Reply:
I don't see where else it's being used 🤔

Reply:
> I don't see where else it's being used 🤔

No, no. My point is, the access key for opening the settings should be one access key used on settings, so when you open settings, the access key for opening settings is disabled, avoiding the possibility of getting an exception.

If you use for example "E", since it's not used on settings, if you open settings and after that you used again the access key "E", it will throw an exception. If the access key is, for example, "O", when you open settings and click again the access key "O" it will not try to open settings, but to open "At startup" page.

Reply:
I'll work on adding proper handling to prevent the exception so that you can use the suggested access key.

Reply:
Ok then, replaced.